### ISSUE TITLE
Slack Lines causing fault issues

### ISSUE THREAD

**Issue by HunterACole** [ChicagoFlyhouse/AutomationConsole#5]
PH 3,11,19 are going into fault when their slack line is triggered.

**Comment by HunterACole** [comment]
There are replacement 1904 cards onsite for swapping to see if that fixes the issue

**Comment by code-warriors-llc** [comment]
This is also I think for PLC team.

**Comment by HunterACole** [comment]
You're correct. We're working on getting PLC team a seat to look at these issues. As of now, I'm using the backend label to indicate which issues I believe are PLC team

**Comment by code-warriors-llc** [comment]
Okay thanks.
